### PR TITLE
Cult's anchor bloodstone's location is now announced in the command alert

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
Hated the search and destroy phase? Want to actually, properly finally know the final place of the bloodstone where the ultimate showdown of ultimate destiny happens? Then this is for you.
This PR has been made by a certified cult fanboy
:cl:
 * tweak: The location of the cult's anchor bloodstone is now announced.

### CODE REVIEW

**Comment by BomberBro** [comment]
Great change, even if cult is losing a lot right now (though only because the meta is calling the shuttle to fight it). The bloodstone phase kinda loses the suspense and potential to be the epic showdown it could be when only the cultists know about the xenobio/perma/maint stone until it's too late.

**Comment by ghost** [comment]
Cult is losing because all the good cult players turned cult off.

**Comment by tuypo1** [comment]
If cultists are gonna get such a major nerf they should be given a way to track their sac targets as compensation

**Comment by boy2mantwicethefam** [comment]
Don't worry I want to add that

**Comment by juanototo2** [comment]
Searching for 3 Port Aft Solars Xenobiology objectives while the opposing force has free stun immunity, bangers but better, and kill'n'stone jugg armies is not very fun. Said objectives also have large health pools, vomit out simple mobs upon taking damage, and losing one provides no tangible benefit until all stones are dead. I would say this slightly reduces the cancer caused by the curbstompy balance philosophy that permeates cult 3.0.

**Comment by ShiftyRail** [comment]
This is purely symbolic because when the anchor actually get triggered you're already fucked in the ass all the way to Sunday
But at least it's a symbol

It's the equivalent of telling the crew where the nuke is anchored when Nuke Ops activated it 10 seconds before the station detonates (and presenting it as a huge nerf, and giving nuke ops a free syndie mecha as a compensation)

@BomberBro cult losing or winning "a lot" shouldn't ever be an argument. We're not at Riot Games HQ. Additionally it's aberrant to try to get statistical arguments when our data sets are completely inhomogenous

**Comment by plosky1** [comment]
Honestly the entire anchor gameplay should just be removed, and instead Nar'sie should have a countdown similar to malf where he will spawn once the countdown ends. Destroying bloodstones should slow that countdown, until the last bloodstone is destroyed ending it entirely.

**Comment by DeityLink** [comment]
Add a 30-60 delay unless there's a powered camera in view of the stone. Other than that I support this completely.

**Comment by ShiftyRail** [comment]
plosky rough draft is already a better idea than the entire current Act IV/V

**Comment by boy2mantwicethefam** [comment]
I'm just gonna add the delay

**Comment by ShiftyRail** [comment]
What difference does it makes

**Comment by boy2mantwicethefam** [comment]
You won't catch me with your typebaiting to slip me you damn clown

**Comment by SirSkeleto** [comment]
people seem to justify things by the worst case and not the common case
if a cult is barely scrounging by and security is huge, i don't think this change increases net fun

**Comment by juanototo2** [comment]
>cult barely scrounging by
>implying they would even get to act IV, not III, IV

**Comment by Tokanova** [comment]
if they're scrounging by, they shouldn't win.

**Comment by boy2mantwicethefam** [comment]
Don't merge this until I make the announcement delayed for about 15-30 seconds

**Comment by boy2mantwicethefam** [comment]
Ok it can be merged

File: code/datums/gamemode/factions/bloodcult/bloodcult.dm [hunk 1]
```diff
@@ -204,8 +205,10 @@ var/veil_thickness = CULT_PROLOGUE
 			var/datum/objective/bloodcult_bloodbath/O = locate() in objective_holder.objectives
 			if (O)
 				veil_thickness = CULT_ACT_IV
-				command_alert(/datum/command_alert/bloodstones_anchor)
 				new_obj = new /datum/objective/bloodcult_tearinreality
+				sleep(150) //A delay before the anchor stone is announced
```
Comment (jknpj) [inline code/datums/gamemode/factions/bloodcult/bloodcult.dm]: SECONDS also why not a spawn?

Comment (jknpj) [reply code/datums/gamemode/factions/bloodcult/bloodcult.dm]: Or a timestamp check

**Pull request merged**